arm/Microsoft.ContainerService/managedClusters/agentPools/readme.md

```diff
@@ -0,0 +1,70 @@
+# ContainerServiceManagedClustersAgentPools `[Microsoft.ContainerService/managedClusters/agentPools]`
```

Comment:
Please update the name to `Managed Cluster AgentPool`

Reply:
Updated the name as requested - used the script this time.